prover/Cargo.lock

```diff
@@ -7880,6 +7882,7 @@ dependencies = [
  "lazy_static",
  "rand 0.4.6",
  "rayon",
+ "regex",
  "reqwest",
  "serde",
  "serde_json",
```

Comment:
I understand you need to update dependencies to be able to use this `debug_basic_circuit` feature? Did you just use newest versions from respective branch (that include bug fixes)? If so, we'd need to regenerate VKs etc?

Reply:
yes I did. And yes, we'll have to regenerate VKs. 

I can put this PR on hold - and we can do the one with new VKs & updated deps first.

Reply:
we should be able to merge it now.